### PR TITLE
Implemented Gaussian Mixture Model in python

### PR DESCRIPTION
**Pull request by vinc3nati**
I have completed the python implementation for GMM for issue #174

### CODE REVIEW

File: Gaussian Mixture Model/GaussianMixtureModel.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        ''' This is the initialization step of the GMM. At this point, we must initialise our parameters  μk, πk and Σk. Here we'll be using results of KMeans as an initial value for μk , πk to one over the number of clusters and Σk to identity matrix. 
+        NOTE: We could also use random numbers for everything, but using a sensible initialisation procedure will help the algorithm achieve better results.
+         '''
+        
+        clusters = []
+        idx = np.arange(X.shape[0])
+        
+        # We use the KMeans centroids to initialise the GMM
+        
+        kmeans = KMeans(self.n_clusters).fit(X)
```
Comment (geekquad) [inline Gaussian Mixture Model/GaussianMixtureModel.py:36]: packages from sklearn are not allowed.

Comment (vinc3nati) [reply Gaussian Mixture Model/GaussianMixtureModel.py:36]: Okay then what alternative can we use ?

**Review by geekquad — APPROVED** [review]
Hey @vinc3nati, good work.

**Comment by vinc3nati** [comment]
> Hey @vinc3nati, good work.

Thank you sir

**Pull request merged**